recipes/humanleague/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "humanleague" %}
+{% set version = "2.0.3" %}
+{% set sha256 = "5dd3744495b8ee72b706e632b6cb9e860a6c7434cdc5b4be4cb4b05ade1f5b67" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py < 34]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
```

Comment:
I believe `python`, `pip` and `numpy` should be moved from the `build` to the `host` section (actually just delete them in `build` and add `numpy` to `host`).